src/gluonnlp/model/bert.py

```diff
@@ -167,108 +94,155 @@ class BERTEncoder(BaseTransformerEncoder):
+            for i in range(num_layers):
+                cell = TransformerEncoderCell(
+                    units=units, hidden_size=hidden_size, num_heads=num_heads,
+                    attention_cell=attention_cell, weight_initializer=weight_initializer,
+                    bias_initializer=bias_initializer, dropout=dropout, use_residual=use_residual,
+                    attention_proj_use_bias=True, attention_use_bias=True, scaled=scaled,
+                    output_attention=output_attention, prefix='transformer%d_' % i,
+                    activation=activation, layer_norm_eps=layer_norm_eps)
+                self.transformer_cells.add(cell)
+
+    def __call__(self, inputs, states=None, valid_length=None): #pylint: disable=arguments-differ
+        """Encode the inputs given the states and valid sequence length.
 
-    Inputs:
-        - **inputs** : input sequence. Shape (batch_size, length, C_in)
-        - **mask** : mask for inputs. Shape (batch_size, length, length)
+        Parameters
+        ----------
+        inputs : NDArray or Symbol
+            Input sequence. Shape (batch_size, length, C_in)
+        states : list of NDArrays or Symbols
+            Initial states. The list of initial states and masks
+        valid_length : NDArray or Symbol
+            Valid lengths of each sequence. This is usually used when part of sequence has
+            been padded. Shape (batch_size,)
+        Returns
+        -------
+        encoder_outputs: list
+            Outputs of the encoder. Contains:
 
-    Outputs:
-        - **outputs**: output tensor of the transformer encoder cell.
-            Shape (batch_size, length, C_out)
-        - **additional_outputs**: the additional output of all the transformer encoder cell.
-    """
+            - outputs of the transformer encoder. Shape (batch_size, length, C_out)
+            - additional_outputs of all the transformer encoder
+        """
+        return super().__call__(inputs, states, valid_length)
+
+    def hybrid_forward(self, F, inputs, states=None, valid_length=None, position_weight=None):
+        # pylint: disable=arguments-differ
+        """Encode the inputs given the states and valid sequence length.
+
+        Parameters
+        ----------
+        inputs : NDArray or Symbol
+            Input sequence. Shape (batch_size, length, C_in)
+        states : list of NDArrays or Symbols
+            Initial states. The list of initial states and masks
+        valid_length : NDArray or Symbol
+            Valid lengths of each sequence. This is usually used when part of sequence has
+            been padded. Shape (batch_size,)
+
+        Returns
+        -------
+        encoder_outputs: list
+            Outputs of the encoder. Contains:
+
+            - outputs of the transformer encoder. Shape (batch_size, length, C_out)
+            - additional_outputs of all the transformer encoder
+
+        Returns
+        -------
```

Comment:
why two returns section?

Reply:
Taken from the `TransformerEncoder`. It's not changed in this PR
https://github.com/dmlc/gluon-nlp/pull/988/files/809187a9437466e7208ec309a608619be48b40c1#diff-411184d223687ccb09d671694e02a8c9L458-L476

Reply:
I didn't notice it before. Let's fix it